app/views/dashboard/_node_moderate.html.erb

```diff
@@ -1,9 +1,10 @@
 <% if node.status == 4 %>
   <p class="alert alert-warning moderated">
     <% if logged_in_as(['admin','moderator']) %>
-      <%= t('dashboard._node_moderate.first_time_post') %> <br class="hidden-xs hidden-sm"/>
+      <%= t('dashboard._node_moderate.first_time_post') %>
       <a class="btn btn-default btn-xs" href="/moderate/publish/<%= node.id %>"><%= t('dashboard.moderate.approve') %></a>
       <a class="btn btn-default btn-xs" href="/moderate/spam/<%= node.id %>"><%= t('dashboard.moderate.spam') %></a>
+      <br class="hidden-xs hidden-sm"/>
```

Comment:
Hey @manansingh96, how about removing the br?

Reply:
I thought about it, but I didn't know why it was there in the first place, so I just changed the order. I will remove it if you say so.

Reply:
I think br is unnecessary here, but let's see. Have you observed any change in the view?

Reply:
Removing `<br/>` doesn't affect the view.

Reply:
Great!